receiver/awscloudwatchmetricsreceiver/README.md

```diff
@@ -0,0 +1,144 @@
+# CloudWatch Metrics Receiver
+
+| Status                   |               |
+| ------------------------ | ------------- |
+| Stability                | [development] |
+| Supported pipeline types | metrics       |
+| Distributions            | [contrib]     |
```

Comment:
Doesn't have to be in this PR, but please follow up with an update to auto-generate the status information